data/js/src/main/scala/sigma/Platform.scala

```diff
@@ -14,33 +14,33 @@ object Platform {
   private[sigma] def liftToConstant(obj: Any, builder: SigmaBuilder): Nullable[Constant[SType]] = {
```

Comment:
Needs case for Option[T]? Based on my understanding you can't do substituteconstants on an Option constant if it's not supported here

Reply:
Seems to be the case indeed, will add

Reply:
Seems better to postpone to 7.0 to minimize codebase changes at this point, made an issue https://github.com/ergoplatform/sigmastate-interpreter/issues/1065